Source/SelfService/Web/api/api.ts

```diff
@@ -168,8 +169,11 @@ export async function saveMicroservice(input: any): Promise<boolean> {
                 'content-type': 'application/json'
             }
         });
-    const jsonResult = await result.json();
-    return true;
+    const jsonResponse = await response.json();
```

Comment:
It might be better to do this after the response.ok, as there is a chance there is no "json()".

Reply:
I'll add a case for that within the `if (!response.ok)`. This case could happen if there is no body to parse into JSON so as to get the message from.

Reply:
Changed it now @freshteapot